autogpt_platform/backend/backend/notifications/conftest.py

```
@@ -0,0 +1,79 @@
+should mock — and mocking it is what keeps them honest.
+
+The suite previously mocked `User.prisma()` directly. That looked equivalent
+and was not: it meant the tests supplied a working database to a process that
+has no connection to one, so a hundred-plus green tests sat on top of a service
+whose scheduled passes failed on every single tick. Mocking here instead means
+a module that reaches past the RPC fails its test rather than passing it.
+"""
+
+from datetime import datetime, timezone
+from types import SimpleNamespace
+from typing import Iterator
```

Ruff (0.16.2)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)